salt/modules/opkg.py

```diff
@@ -582,6 +591,12 @@ def install(name=None,
             info={'errors': errors, 'changes': ret}
         )
 
+    if list_pkgs_errors:
+        raise CommandExecutionError(
+            'Problem encountered after installing package(s). Cannot provide changes list.',
```

Comment:
Can we be even more specific and point out that the installation was with success? "Problem encountered after installing package(s) with success". What do you think?

Reply:
"after succesfully installing package(s)" sounds good?